packages/components/atoms/f-error-boundary/src/components/tests/ErrorBoundary.test.js

```diff
@@ -0,0 +1,140 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import ErrorBoundary from '../ErrorBoundary.vue';
+
+const localVue = createLocalVue();
+
+const createComponent = ({
+    propsData,
+    computed,
+    scopedSlots
+} = {}) => shallowMount(ErrorBoundary, {
+    localVue,
+    computed,
+    propsData,
+    scopedSlots
+});
+
+describe('ModalDeliveryFeesInfo', () => {
```

Comment:
Should this be something like `describe('ErrorBoundary', () => {` ?

Reply:
It should be, yes. Fixed now 👍🏻